inst/extdata/var_src_codes.tsv

```diff
@@ -38,7 +38,7 @@ suntime	calcLon	''	apparent solar time	ts	local.time	1	NA	calculate suntime from
 suntime	simCopy	''	apparent solar time	ts	local.time	12	NA	simulate depth by copying from another site	streamMetabolizer	NA	sim	NA
 suntime	simLon	''	apparent solar time	ts	local.time	11	NA	calculate suntime from UTC and site longitude	streamMetabolizer	convert_GMT_to_solartime	sim	NA
 suntime	simNew	''	apparent solar time	ts	local.time	13	NA	arbitrarily make up numbers for model testing	streamMetabolizer	NA	sim	NA
-sw	nldas	?	downwelling shortwave radiation	ts	NA	1	http://ldas.gsfc.nasa.gov/nldas/	Land Data Assimilation Systems	geoknife	geoknife	data	dswrfsfc
+sw	nldas	W/m^2	downwelling shortwave radiation	ts	NA	1	http://ldas.gsfc.nasa.gov/nldas/	Land Data Assimilation Systems	geoknife	geoknife	data	dswrfsfc
```

Comment:
at some point...maybe soon...we'll want to be storing units in unitted format, i.e., "W m^-2"